### PR TITLE
2.0.0

### PR DESCRIPTION
**Pull request by syedalinaqi**
Can u also add icons from http://www.entypo.com/

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,69 +1,90 @@
...
+The base library contains more than XXX icons, and allows you to add your own icons from icon font generation services like [fontello.com](http://fontello.com/), [icomoon.io](https://icomoon.io/) or [fontastic.me](http://fontastic.me/).
 
 -----
 
-### About
+### Install
+
+You don't need to import all the modules, choose the modules you'll want to use.
+
+```gradle
```
Comment (syedalinaqi) [inline README.md]: why cant i see them on the main page???

**Comment by JoanZapata** [comment]
What makes you think I want to merge this branch right now?

File: README.md (hunk 1 above)
Comment (JoanZapata) [inline README.md]: This branch is a work in progress.

**Comment by JoanZapata** [comment]
(Entypo support was added just now BTW)

**Pull request closed without merging**